client/views/windows/settings.tpl

```diff
@@ -28,6 +28,10 @@
 			</label>
 			<p class="sync-warning-override"><strong>Warning</strong> Checking this box will override the settings of this client with those stored on the server.</p>
 			<p class="sync-warning-base"><strong>Warning</strong> No settings have been synced before. Enabling this will sync all settings of this client as the base for other clients.</p>
+			<label class="opt force-sync-button">
+				<button type="button" class="btn" id="forceSync">Force sync settings</button><br>
```

Comment:
I feel like this is more commonly called "Set/Mark as default", but it's not perfect either. I just find "Force sync settings" not ideal.

I'm overall very confused by the wording of the feature, but I'm not sure how to make it clearer just now. Worth waiting for the feature to stabilize and see if we can improve.

<img width="469" alt="screen shot 2018-04-02 at 02 22 24" src="https://user-images.githubusercontent.com/113730/38185559-b8d79e08-361c-11e8-8df5-e6179c50471c.png">

Good thing it's an advanced feature :D

Reply:
I agree the wording isn't ideal but as you pointed out neither is other wording. And yeah this is a proper advanced setting only used for those people that already have settings they are particularly attached to.